cmd/generate/config/rules/notion.go

```diff
@@ -0,0 +1,38 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Notion() *config.Rule {
+    // Define the identifiers that match the Keywords
+    identifiers := []string{"ntn", "ntn_", "notion"}
+    
+    // Define the regex pattern for Notion API token
+    secretRegex := `ntn_[0-9]{11}[A-Za-z0-9]{32}[A-Za-z0-9]{3}`
+    
+    regex := utils.GenerateUniqueTokenRegex(secretRegex, false)
+    
+    r := config.Rule{
+        Description: "Notion API token",
+        RuleID: "notion-api-token",
+        Regex: regex,
+        Keywords: identifiers,
```

Comment:
[Entropy](https://lookingatcomputer.substack.com/p/regex-is-almost-all-you-need) is essential for filtering low-quality matches (e.g., `ntn_00000000000aaaaaaaaaaaaaaaaa...`). Using the [Gitleaks playground](https://gitleaks.io/playground) I can see that most tokens have an entropy > 4.5, so 4 seems like a reasonable lower bound. 
```suggestion
        Regex: regex,
        Entropy: 4,
        Keywords: identifiers,
```